### PR TITLE
Add pytest-chalice

### PR DESCRIPTION
**Pull request by oblute**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@benhuff @mwilson8 do you agree to be maintainers?

**Comment by oblute** [comment]
@conda-forge/help-python @conda-forge/staged-recipes this recipe is ready for your review

File: recipes/pytest-chalice/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  sha256: 1cf7ba41970d574315e5a2ae241476046b2fc674ce7d38d307560ddb4e433d3a
+  patches:
+    - setuptools_scm.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - chalice = pytest_chalice.plugin:main_entry_point
+  skip: True  # [win or py27]
```
Comment (scopatz) [inline recipes/pytest-chalice/meta.yaml]: Is there any way that you can make this noarch python? You can use the `python >=35` in the requirements sections

Comment (oblute) [reply recipes/pytest-chalice/meta.yaml]: @scopatz works as noarch - added it in with the python pinnings.

**Comment by oblute** [comment]
@scopatz should be good to go now

**Comment by scopatz** [comment]
great thanks!

**Pull request merged**